validation-test/stdlib/SequenceType.swift.gyb

```diff
@@ -617,6 +617,26 @@ SequenceTypeTests.test("contains/Predicate") {
 }
 
 //===----------------------------------------------------------------------===//
+// first()
+//===----------------------------------------------------------------------===//
+
+SequenceTypeTests.test("first/Predicate") {
```

Comment:
I'll move it. It still isn't quite clear to me exactly where the different kinds of tests belong. Is that stuff documented anywhere?